central/compliance/standards/index/indexer.go

```diff
@@ -1,83 +1,30 @@
 package index
 
 import (
-	"github.com/blevesearch/bleve"
+	"context"
+
 	v1 "github.com/stackrox/rox/generated/api/v1"
-	"github.com/stackrox/rox/pkg/logging"
 	"github.com/stackrox/rox/pkg/search"
 	"github.com/stackrox/rox/pkg/search/blevesearch"
 )
 
 var (
-	log = logging.LoggerForModule()
-
 	// StandardOptions is the search options map for a compliance standard
 	StandardOptions = search.Walk(v1.SearchCategory_COMPLIANCE_STANDARD, "standard", (*v1.ComplianceStandard)(nil))
 	// ControlOptions is the search options map for a compliance control
 	ControlOptions = search.Walk(v1.SearchCategory_COMPLIANCE_CONTROL, "control", (*v1.ComplianceControl)(nil))
 )
 
-// Indexer is the interface for Compliance Search
+// StandardIndexer implements the indexer for standards
 //
 //go:generate mockgen-wrapper
-type Indexer interface {
-	IndexStandard(standard *v1.ComplianceStandard) error
-	DeleteStandard(id string) error
-	DeleteControl(id string) error
-	SearchStandards(q *v1.Query) ([]search.Result, error)
-	SearchControls(q *v1.Query) ([]search.Result, error)
-}
-
-type controlWrapper struct {
-	*v1.ComplianceControl `json:"control"`
-	Type                  string `json:"type"`
-}
-
-type standardWrapper struct {
-	*v1.ComplianceStandard `json:"standard"`
-	Type                   string `json:"type"`
+type StandardIndexer interface {
+	Search(ctx context.Context, q *v1.Query, opts ...blevesearch.SearchOption) ([]search.Result, error)
```

Comment:
Do we still need to use `blevesearch` options?  Can we or are we going to move them elsewhere?